packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolylineController_Test.h

```diff
@@ -11,14 +11,3 @@
 @property(strong, nonatomic) GMSPolyline *polyline;
 
 @end
-
-/// Internal APIs explosed for unit testing
-@interface FLTPolylinesController (Test)
-
-/// Returns the path for polyline based on the points(locations) the polyline has.
-///
-/// @param polyline The polyline instance for which path is calculated.
-/// @return An instance of GMSMutablePath.
-+ (GMSMutablePath *)pathForPolyline:(NSDictionary *)polyline;
```

Comment:
This is one of the things replaced by `FGMGetPathFromPoints`, so is tested that way instead.